tests/nodeos_extra_packed_data_test.py

```diff
@@ -19,34 +19,29 @@
 cmdError=Utils.cmdError
 
 args = TestHelper.parse_args({"--host","--port","-p","--defproducera_prvt_key","--defproducerb_prvt_key"
-                              ,"--dump-error-details","--dont-launch","--keep-logs","-v","--leave-running","--clean-run"
+                              ,"--dump-error-details","--dont-launch","--keep-logs","-v","--leave-running"
                               ,"--sanity-test","--wallet-port","--unshared"})
 server=args.host
 port=args.port
 debug=args.v
 defproduceraPrvtKey=args.defproducera_prvt_key
 defproducerbPrvtKey=args.defproducerb_prvt_key
 dumpErrorDetails=args.dump_error_details
-keepLogs=args.keep_logs
 dontLaunch=args.dont_launch
-dontKill=args.leave_running
 pnodes=args.p
-killAll=args.clean_run
 sanityTest=args.sanity_test
 walletPort=args.wallet_port
 
 Utils.Debug=debug
 localTest=True if server == TestHelper.LOCAL_HOST else False
 cluster=Cluster(host=server, 
                 port=port, 
-                walletd=True,
                 defproduceraPrvtKey=defproduceraPrvtKey, 
                 defproducerbPrvtKey=defproducerbPrvtKey,
-                unshared=args.unshared)
+                unshared=args.unshared,
+                keepRunning=args.leave_running, keepLogs=args.keep_logs)
```

Comment:
Seems like the style in this file is one arg per line.

Reply:
Revised.

Reply:
5f3a8d7d8